packages/ui/src/components/va-tree-view/components/VaTreeNode/VaTreeNode.vue

```diff
@@ -241,6 +246,10 @@ export default defineComponent({
     }
   }
 
+  &--cursor {
+    cursor: var(--va-tree-node-cursor-type);
```

Comment:
```suggestion
    cursor: pointer;
```

I find strange user for typing code like:
```css
:root {
  --va-tree-node-cursor-type: loading;
}
```
99% user want to have cursor pointer over interactive (expandable) element. It is also unclear from CSS variable that this variable is used for only nodes which can be expanded.